packages/config-utils/extensions-plugin.js

```diff
@@ -0,0 +1,54 @@
+const { resolve } = require('path');
+const fedModule = require('./federated-modules');
+const ExtensionsMapper = require('./extension-mapper');
+
+class ExtensionsPlugin {
+    constructor(plugin, fedMod, options) {
+        if (!plugin) {
+            throw new Error('Missing plugin config!');
+        }
+
+        this.plugin = plugin;
+        this.fedMod = {
+            moduleName: 'plugin-entry',
+            libName: 'window.loadPluginEntry',
+            useFileHash: false,
+            libType: 'jsonp',
```

Comment:
I am wondering if this will cause issues. Is it correct that the extensions are built with webpack 4? If so, I am afraid it can cause conflicts in the webpack namespace. Or are the CRC extension have a separate build using webpack 5?

Reply:
Will you be testing the plugin before merging this PR? Or do you want to just do a "silent" release?

Reply:
I've managed to run it, let's merge it!